packages/api-catalog-spa/src/pages/ApiCUDPage/components/EnvironmentFormSection/EnvironmentFormSection.tsx

```diff
@@ -98,6 +98,7 @@ export const EnvironmentFormSection = (): JSX.Element => {
                           placeholder="Enter env name"
                           isCreatable
                           onCreateOption={onEnvNameCreate}
+                          placeholderText="Enter env name"
```

Comment:
```suggestion
                          placeholderText="Enter environment name"
```

Reply:
ENV in a tech shothand use full name to understand it

Reply:
Done